tests/testsuite/cargo_install/help/stdout.term.svg

```diff
@@ -35,103 +35,105 @@
 </tspan>
     <tspan x="10px" y="154px"><tspan class="fg-green bold">Options:</tspan>
 </tspan>
-    <tspan x="10px" y="172px"><tspan>      </tspan><tspan class="fg-cyan bold">--version</tspan><tspan class="fg-cyan"> </tspan><tspan class="fg-cyan">&lt;VERSION&gt;</tspan><tspan>     Specify a version to install</tspan>
+    <tspan x="10px" y="172px"><tspan>  </tspan><tspan class="fg-cyan bold">-n</tspan><tspan>, </tspan><tspan class="fg-cyan bold">--dry-run</tspan><tspan>               Perform all checks without installing (unstable)</tspan>
```

Comment:
`cargo test` should pass on every commit.

Reply:
I've moved this in the same commit as the one who add the flag.